results/index.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: page
+notitle: true
+title: Results
+datatable: true
+---
+
+# MLBench Official Training Results
+---
+
+<p>
+Here we present the official training results obtained by running the benchmark tasks.
+It provides a reference and comparison point for different implementations.
+</p>
+
+
+<p>
+The table below provides a comparison table for results. Times are in seconds. The baseline represents the 1 worker case, and we
+only present the results for the best obtained speedups.
+
+For more detailed results for each run, please check the documentation.
+
+</p>
+
+---
+
+|Task                 | Dataset  | Model                 | Aggregation scheme |  Metric Name           | Metric Goal | Framework    |    Baseline    | Best Speedup   | Best Speedup Workers | Description | Implementation | Results |
```

Comment:
instead of best speedup, let's only put the speedup at the max (or reasonable, fixed) number of workers, so 16?
then you can have 2 columns, one for total speedup (most important), and if you want a second one for compute only speedup. (and a column to say that #workers = 16)

and Bleu scores can have a decimal for clarity. 24.0 and 25.0  (minor nitpick :))

Reply:
if other experiments have a different max # workers that's ok too